src/traces/carpet/axis_attributes.js

```diff
@@ -0,0 +1,445 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var extendFlat = require('../../lib/extend').extendFlat;
+var fontAttrs = require('../../plots/font_attributes');
+var colorAttrs = require('../../components/color/attributes');
+
+module.exports = {
+    color: {
+        valType: 'color',
+        role: 'style',
+        description: [
+            'Sets default for all colors associated with this axis',
+            'all at once: line, font, tick, and grid colors.',
+            'Grid color is lightened by blending this with the plot background',
+            'Individual pieces can override this.'
+        ].join(' ')
+    },
+    smoothing: {
+        valType: 'number',
+        dflt: 1,
+        min: 0,
+        max: 1,
```

Comment:
it would be nice to make this consistent with the `scatter` definition of [`smoothing`](https://github.com/plotly/plotly.js/blob/6e547ef794093c206998103f3065d5f2ab2a20b4/src/traces/scatter/attributes.js#L154) where the default value is `1` and the max `1.3`.